.github/workflows/test.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       # https://github.com/pnpm/action-setup
       - name: Install pnpm
-        uses: pnpm/action-setup@v2
+        uses: pnpm/action-setup@v4
```

Comment:
The previous action versions were deprecated and did not support new versions of `node`.